clang/include/clang/AST/DeclTemplate.h

```diff
@@ -2085,7 +2085,9 @@ class ClassTemplateSpecializationDecl : public CXXRecordDecl,
 class ClassTemplatePartialSpecializationDecl
   : public ClassTemplateSpecializationDecl {
   /// The list of template parameters
-  TemplateParameterList* TemplateParams = nullptr;
+  TemplateParameterList *TemplateParams = nullptr;
+
+  TemplateArgument *InjectedArgs = nullptr;
```

Comment:
Do we need serialization?

Reply:
This is just for catching the injected arguments. If we don't serialize these, they will be just rebuilt anyway.

Reply:
Yeah, like @mizvekov said we don't serialize injected template arguments